### PR TITLE
Feat/hii 3862 optimize yaml definitions

### PR DESCRIPTION
**Pull request by DStadnyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,14 +193,150 @@ developer-portal 📂
...
+└───Product 2 📂
+    │  
+    │  ...
+```
+
+### Private docs
+
+Developer portal allows you creating private docs. These docs will only be visible to authenticated Extenda users.
+
+Use private docs to explain architecture,
```
Comment (l0kal) [inline README.md:278]: This is not a full sentence :smile:
Are you missing the information, or do you not have the information needed to write the paragraph?

File: README.md [hunk 2]
```diff
@@ -193,14 +193,150 @@ developer-portal 📂
...
+│       │    │  
+│       │    └───api 📂
+│       │    │    │    
+│       │    │    │  api.yaml
+│       │    │    │     
+│       │    │    └───docs 📂
+│       │    │    │    doc1.md
+│       │    │    │    doc2.md
+│       │    │    │    ...     
+│       │    │
```
[the remark this answers is not in the document]
Comment (DStadnyk) [reply README.md]: Good catch, fixed it

File: README.md (hunk 1 above)
Comment (DStadnyk) [reply README.md]: Fixed it 🤣

File: README.md [hunk 3]
```diff
@@ -193,14 +193,150 @@ developer-portal 📂
...
+
+### Private docs
+
+Developer portal allows you creating private docs. These docs will only be visible to authenticated Extenda users.
+
+Use private docs to explain architecture, 
+
+#### Private pages
+
+If a page doesn't have any specific permission, that means the page will use the default permission from the rbac.yaml file. By default, all pages are publicly accessible.
```
[the remark this answers is not in the document]
Comment (DStadnyk) [reply README.md:301]: `permissions.rbac.yaml` explained on [this](https://github.com/extenda/dev-portal-common-repo-docs-template/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R322) line

[the remark this answers is not in the document]
Comment (DStadnyk) [reply README.md:301]: And permission for private page explained [here](https://github.com/extenda/dev-portal-common-repo-docs-template/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R308)

File: README.md [hunk 4]
```diff
@@ -193,14 +191,171 @@ developer-portal 📂
...
+│       │    │    │    doc1.md
+│       │    │    │    doc2.md
+│       │    │    │    ...     
+│       │    │    
+│       │    └───images 📂
+│       │    │    │    img1.png
+│       │    │    │    img2.png
+│       │    │    │    ...   
+│       │    │  
+│       │    └───api 📂
```
Comment (l0kal) [inline README.md:251]: This should be `apis` in plural

**Review by l0kal — APPROVED** [review]

**Pull request merged**